python/src/aac/lang/language_context.py

```diff
@@ -419,6 +437,17 @@ def add_plugins(self, plugins: list[Plugin]):
         """Add the specified plugins to the current language context."""
         self.plugins.extend(plugins)
 
+    def add_named_plugins(self, names: list[str]):
+        """
+        Add the plugin from the provided file URI.
+
+        Args:
+            names (list[str]): The names of the plugins which should be loaded. If not provided, all
+                plugins will be loaded.
+        """
+        plugins = [plugin for plugin in plugin_manager.get_plugins() if plugin.name in names]
+        self.add_plugins(plugins)
+
```

Comment:
This will need to be de-conflicted with https://github.com/jondavid-black/AaC/pull/499/files#diff-d52e144e2a86781dcfbc2e77b4ee4f8da334d641b1f37713014eec5c9a729f67R427-R430

Reply:
Yeah, I noticed that yesterday.